packages/flutter/lib/src/material/app.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// @docImport 'package:flutter_localizations/flutter_localizations.dart';
```

Comment:
I didn't know we could import this? Does that mean/require the material library has a dependency on `flutter_localizations` defined in the framework's `pubspec.yaml`?

Reply:
Yeah, I added a **dev** dependency for it so it could be used in docimports without affecting our downstream users.